### ISSUE TITLE
Consistency in y/n prompts

### ISSUE THREAD

**Issue by carlynorama** [swiftlang/swiftly#141]
Some accept just y or just Y some both. (swifty-install.sh, the intro to the package accepts both.)

MacOS.swift and Linux.swift goes their own ways in asking for confirmation (line 121, 135 respectively) in their `use()` functions, only accepting y, but not warning on a Y. 

perhaps these functions should be using 

`promptForConfirmation(defaultBehavior: Bool)` from Utils.swift (line 19)? 

it seems like swiftly can end up in a pretty bad state after an aborting a use if the symlinks have already been changed (hold please, currently researching) but I got there by typing Y instead of y.

**Comment by patrickfreed** [comment]
Yeah we should definitely be consistent with this. Merging `swiftly-install.sh` into swiftly itself will help with that too, we just need to make sure to use the same `promptForConfirmation` everywhere once we do that.

**Comment by adam-fowler** [comment]
@cmcgee1024 Can you verify your `swiftly init` PR is consistent
